### DISCUSSION TITLE
video error when open Google chrome in client

### DISCUSSION THREAD

**Discussion by doguibnu** [ltsp/ltsp#494] [General]
Hello!

My LTSP server has Xubuntu 20.04 LTS
Ltsp server ok and working but, when one client machine open google chrome, freeze system and shows screen as attached file.
Yes, the client is an old machine 
Look
![ltsp-error](https://user-images.githubusercontent.com/38897311/125165757-c2748380-e16e-11eb-82b8-3a3282132054.png)

Is there solution?

Thnaks

Douglas

**Comment by alkisg** [comment]
What are the client exact specifications?
https://ltsp.org/advanced/specs/

Try to boot the client with a live CD/USB.
If the problem happens there too, then the problem isn't related to LTSP and you can ask about it in #ubuntu etc.
